docs/keps/20200619-federated-resource-status.md

```diff
@@ -189,21 +260,35 @@ The value of `status.conditions` should reflect the new state for that specific
   status:
     clusters:
     - name: "cluster3"
-      status: "ReplicaFailure"
+      status: "CreationFailed"
+      conditions:
+      - lastTransitionTime: "2020-05-25T20:23:59Z"
+        lastUpdateTime: "2020-05-25T20:23:59Z"
+        status: "True"
+        type: "NotReady"
+        reason: "ReplicaFailure"
+        message: "1 Replica is not ready, you don't have enough resources"
     conditions:
     - lastTransitionTime: "2020-05-25T20:23:59Z"
       lastUpdateTime: "2020-05-25T20:23:59Z"
-      status: "True"
-      type: "NotReady"
+      status: "False"
+      type: "Propagation"
 ```
 
-The value of `spec.conditions` contains a `NotReady` condition type, that is the
-result of checking the status of that remote `Deployment` in the target clusters.
-The value of `status.clusters.status` can be extracted from the `"ReplicaFailure"`
-status of the `Deployment` and so reused for visibility.
+The value of `status.conditions` describes the propagation error that happened to `cluster3`
+while `clusters["cluster3"].conditions` contains a `NotReady` condition type with a detailed
+description of the state of the federated resource in `cluster3`.
+The value of `status.clusters["cluster3"].status` reuses the current set of available status types, e.g. `"CreationFailed"`, which identifies the current problem during the propagation.
+
 
-Likewise, the status of the `FederatedDeployment` remains `Ready=True` in the rest
+The status of the `FederatedDeployment` remains `Ready=True` in the rest
 of clusters: `cluster1` and `cluster2`.
+However as you can see, the `clusters[*].conditions` does not contain values with the `IsReady` condition
```

Comment:
Are we going to support regular expressions?

Reply:
No, I'll replace `[*]` by `[<index>]`.